src/components/NavBar/ChainSelector.tsx

```diff
@@ -72,6 +62,9 @@ export const ChainSelector = ({ leftAlign }: ChainSelectorProps) => {
 
   const theme = useTheme()
 
+  const showTestnets = useAtomValue(showTestnetsAtom)
+  const chains = NETWORK_SELECTOR_CHAINS.filter((chain) => !(TESTNET_SET.has(chain) && !showTestnets))
```

Comment:
sry one more nit - this is kinda confusing to me, wdyt 

`const chains = showTestnets ? NETWORK_SELECTOR_CHAINS : NETWORK_SELECTOR_CHAINS.filter(chain => !TESTNET_SET.has(chain))`

Reply:
```suggestion
  const chains = showTestnets ? NETWORK_SELECTOR_CHAINS : NETWORK_SELECTOR_CHAINS.filter(chain => !TESTNET_SET.has(chain))
```